CONTRIBUTING.md

```diff
@@ -108,6 +108,32 @@ The version-specific directory names have one of the following forms / use cases
 - `src/main/302to312-cdh` contains code that applies to Cloudera CDH shims between 3.0.2 *inclusive*,
    3.1.2 *inclusive*
 
+
+### Setting up an Integrated Development Environment
+
+Our project currently uses `build-helper-maven-plugin` for shimming against conflicting definitions of superclasses 
+in upstream versions that cannot be resolved without significant code duplication otherwise. This may require some 
+modifications to IDEs' standard Maven import functionality.
+
+#### IntelliJ IDEA
+
+_Last tested with 2021.2.1 Community Edition_
+
+To start working with the project in IDEA is as easy as 
+[opening](https://blog.jetbrains.com/idea/2008/03/opening-maven-projects-is-easy-as-pie/) the top level (parent) 
+[pom.xml](pom.xml). 
+
```

Comment:
For me, it was also necessary to run one `mvn package -DskipTests` before I could run any tests. Without this, tests failed with `Could not find properties file rapids4spark-version-info.properties in the classpath`.

Reply:
I believe this is a different IDE-unrelated regression #3461 that was introduced but I'll add wording about it for now if this workaround works consistently.

Reply:
I think we'll have to defer to #3461, I am not sure if this works generally. @andygrove feel free to add your instructions in a separate PR with the exact steps for your workaround.